crates/bevy_render/src/render_resource/uniform_buffer.rs

```diff
@@ -42,7 +43,8 @@ impl<T: ShaderType> From<T> for UniformBuffer<T> {
             scratch: UniformBufferWrapper::new(Vec::new()),
             buffer: None,
             label: None,
-            label_changed: false,
+            changed: false,
+            buffer_usage: BufferUsages::COPY_DST | BufferUsages::UNIFORM,
```

Comment:
These defaults need to be documented, and the latter use of`BufferUsages::UNIFORM` should be strictly enforced.